recirq/qaoa/problem_circuits_test.py

```diff
@@ -42,38 +42,38 @@ def test_get_routed_grid_model_circuit():
     )
 
     cirq.testing.assert_has_diagram(circuit, """
-  (0, 0)   (0, 1)   (0, 2)   (1, 0)   (1, 1)   (1, 2)
-  │        │        │        │        │        │
-  H        H        H        H        H        H
-  │        │        │        │        │        │
-  ZZ───────ZZ       │        ZZ───────ZZ       │
-  │        │        │        │        │        │
-  │        ZZ───────ZZ       │        ZZ───────ZZ
-  │        │        │        │        │        │
-┌╴│        │        │        │        │        │       ╶┐
-│ ZZ───────┼────────┼────────ZZ       │        │        │
-│ │        ZZ───────┼────────┼────────ZZ       │        │
-│ │        │        ZZ───────┼────────┼────────ZZ       │
-└╴│        │        │        │        │        │       ╶┘
-  │        │        │        │        │        │
-  │        │        │        │        │        │
-  │        │        │        │        │        │
-  Rx(π)    Rx(π)    Rx(π)    Rx(π)    Rx(π)    Rx(π)
-  │        │        │        │        │        │
-  ZZ───────ZZ^0.5   │        ZZ───────ZZ^0.5   │
-  │        │        │        │        │        │
-  │        ZZ───────ZZ^-0.5  │        ZZ───────ZZ^0.5
-  │        │        │        │        │        │
-┌╴│        │        │        │        │        │       ╶┐
-│ ZZ───────┼────────┼────────ZZ^-0.5  │        │        │
-│ │        ZZ───────┼────────┼────────ZZ^0.5   │        │
-│ │        │        ZZ───────┼────────┼────────ZZ^0.5   │
-└╴│        │        │        │        │        │       ╶┘
-  │        │        │        │        │        │
-  │        │        │        │        │        │
-  │        │        │        │        │        │
-  Rx(0.5π) Rx(0.5π) Rx(0.5π) Rx(0.5π) Rx(0.5π) Rx(0.5π)
-  │        │        │        │        │        │
+  (0, 0) (0, 1) (0, 2)  (1, 0)  (1, 1) (1, 2)
+  │      │      │       │       │      │
+  H      H      H       H       H      H
+  │      │      │       │       │      │
+  ZZ─────ZZ     │       ZZ──────ZZ     │
+  │      │      │       │       │      │
+  │      ZZ─────ZZ      │       ZZ─────ZZ
+  │      │      │       │       │      │
+┌╴│      │      │       │       │      │     ╶┐
+│ ZZ─────┼──────┼───────ZZ      │      │      │
+│ │      ZZ─────┼───────┼───────ZZ     │      │
+│ │      │      ZZ──────┼───────┼──────ZZ     │
+└╴│      │      │       │       │      │     ╶┘
+  │      │      │       │       │      │
+  │      │      │       │       │      │
+  │      │      │       │       │      │
+  X      X      X       X       X      X
```

Comment:
annoyingly, this test behavior will now change depending on cirq version that you're using recirq with, although the circuits have the same effect.

Reply:
Optional:  Is there a better way to test it besides testing the diagram output?  If not, oh well I guess?

Reply:
I'll investigate. I think we have some tests for unitary equivalence, which is the important thing. This may have been an additional test to be sure (or to check for keeping the circuit structured how we expected)